### PR TITLE
Rename Eswatini from Swaziland

### PR DESCRIPTION
**Pull request by bkthomps**
This country has a different name now (but keep it backwards-compatible).

### CODE REVIEW

File: holidays/countries/__init__.py [hunk 1]
```diff
@@ -89,7 +89,7 @@
 from .slovenia import Slovenia, SI, SVN
 from .south_africa import SouthAfrica, ZA, ZAF
 from .spain import Spain, ES, ESP
-from .swaziland import Swaziland, SZ, SZW
+from .eswatini import Eswatini, Swaziland, SZ, SZW
```
Comment (arkid15r) [inline holidays/countries/__init__.py:92]: It seems that the imports need to be sorted alphabetically.

**Review by arkid15r — APPROVED** [review]
LGTM

File: holidays/countries/__init__.py (hunk 1 above)
Comment (bkthomps) [reply holidays/countries/__init__.py]: Done.

**Comment by dr-prodigy** [comment]
Merged.. Thx!

**Pull request merged**